### PR TITLE
Fix for VO truncation on tests with repeated key presses

### PR DESCRIPTION
**Pull request by ChrisC**
fixes https://github.com/w3c/aria-at/issues/1164

Adds a general microdelay after every keypress to account for cases when a keypress is repeated in a test (for example: Assertion Results for "Control+Option+Left Arrow, then Control+Option+Left Arrow, then Control+Option+Left Arrow" in the Disclosure pattern tests).

VO output now consistently matches the expected output described by manual testers in https://github.com/w3c/aria-at/issues/1164

### CODE REVIEW

**Review by stalgiag — COMMENTED** [review]
Makes sense! Our applescript will be a lot gnarlier but no one sees that :-P

In my testing, the Shift delay fix seemed to work with *any* value for the delay. This is just to say that I think your general delay solution could be used for the shift key as well. No need for a specific carve out for that specific delay duration.

File: packages/macos-at-driver-server/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bocoup/macos-at-driver-server",
-  "version": "0.0.7",
+  "version": "0.0.8",
```
Comment (jugglinmike) [inline packages/macos-at-driver-server/package.json:3]: Although we intend to release your fix immediately in this case, I'd still prefer to do this with a dedicated commit.

```suggestion
  "version": "0.0.7",
```

File: shared/helpers/macos/applescript.js [hunk 2]
```diff
@@ -22,6 +22,9 @@ const APPLESCRIPT_TIMEOUT = 5;
 /** Error message to look for to determine error is an Applescript time out. */
 const APPLESCRIPT_TIMEOUT_ERROR = (exports.APPLESCRIPT_TIMEOUT_ERROR = 'AppleEvent timed out');
 
+const SHIFT_CHORD_DELAY = 0.05;
+const KEYPRESS_DELAY = 0.25;
```
Comment (jugglinmike) [inline shared/helpers/macos/applescript.js:26]: I somehow thought I'd have to ask you to pull these out as constants, but of course you're on top of it :)

**Review by jugglinmike — APPROVED** [review]
Looks good to me! Just a small change related to the release process

**Comment by ChrisC** [comment]
> Looks good to me! Just a small change related to the release process

Cool just got that change in. I think once the tests pass this will be ready to merge and publish.

**Comment by jugglinmike** [comment]
Cool. We had a spurious failure, so I re-ran the job, and everything looks good, now. I'll follow up with a new release right away
